pkg/mermaid/mermaidsequencediagram.go

```diff
@@ -0,0 +1,157 @@
+package mermaid
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+
+	"github.com/anz-bank/sysl/pkg/sysl"
+)
+
+type sequencepair struct {
+	firstapp, secondapp, endpoint string
+}
+
+const projDir = "../../"
+
+var startElse = regexp.MustCompile("^else.*")
+var isElse = regexp.MustCompile("^else$")
+var isLoop = regexp.MustCompile("^loop.*")
+
+func GenerateMermaidSequenceDiagram(m *sysl.Module, appname string, epname string) (string, error) {
```

Comment:
Public comments would be nice (I also think private comments would be nice, but that's upto you)

Reply:
Agreed!